expression/expression_test.go

```diff
@@ -19,82 +20,137 @@ func TestExpression(t *testing.T) {
 	Convey("Test Default", t, func() {
 		warnValue := 60.0
 		errorValue := 90.0
-		result, err := (&TriggerExpression{MainTargetValue: 10.0, WarnValue: &warnValue, ErrorValue: &errorValue, TriggerType: moira.RisingTrigger}).Evaluate()
-		So(err, ShouldBeNil)
+		expression := &TriggerExpression{MainTargetValue: 10.0, WarnValue: &warnValue, ErrorValue: &errorValue, TriggerType: moira.RisingTrigger}
```

Comment:
As if you could not validate an expression from Simple Mode, because it will always be correctly written, the problem may be only in the calculation, but this is already a `gavaluate` task

Reply:
I believe that this test was added to prevent breaking changes, i added the call to Validate() to make sure that simple expressions have a 100% guarantee to be safe as well. I can remove it if you want.